cmd/openshift-install/create.go

```diff
@@ -288,11 +286,9 @@ func destroyBootstrap(ctx context.Context, config *rest.Config, directory string
 		return errors.Wrap(err, "waiting for Kubernetes API")
 	}
 
-	events := client.CoreV1().Events("kube-system")
-
 	eventTimeout := 30 * time.Minute
 	logrus.Infof("Waiting up to %v for the bootstrap-complete event...", eventTimeout)
-	if err := waitForEvent(ctx, events, "bootstrap-complete", eventTimeout); err != nil {
+	if err := waitForEvent(ctx, client.CoreV1().RESTClient(), "event", "kube-system", "bootstrap-complete", eventTimeout); err != nil {
```

Comment:
Also the resource is `events`

Reply:
```console
$ oc get --raw /api/v1/ | jq '.resources[].name' | grep events
"events"
```

@sttts might have a better way to verify.

Reply:
> Also the resource is `events`

And I'm missing the typed client more already :p.  Fixed with 30e79f40d -> de62f647a.